src/components/plugins/AddPluginDialog.tsx

```diff
@@ -88,7 +87,7 @@ interface Props {
   projects: Project[] | null;
   services: ServiceFunctions;
   setError: (error: string) => void;
-  getPlugins: () => Promise<void>;
+  getPlugins: () => void;
```

Comment:
Why no longer a Promise?

Reply:
I've changed `getPlugins` (defined in `PluginsView`) from an async to a sync method, by replacing async-await with then-catch to call that method in a fire-and-forget way.